lib/database/classes.ts

```diff
@@ -77,5 +81,24 @@ const getClass = async (id: string): Promise<Class | null> => {
 
   return classes;
 };
+const getAllClasses = async (): Promise<Class[]> => {
+  const query = {
+    text: "SELECT e.name, c.event_information_id, c.min_level, c.max_level, c.rrstring, c.start_time, c.end_time, c.language FROM event_information e, classes c WHERE e.id = c.event_information_id AND e.type = 'class'",
```

Comment:
In the database, these are being saved as a "Class" not "class". So select where type='Class'